### PR TITLE
Close Issue

### PR DESCRIPTION
**Pull request by helloqiu**
Hi there.
I create a bot to help developers to **close issues that do not include some specific contents**, which is useful when **the developers provide issue template but the issue does not refer to it**.
I create the bot because I don't find other bots that have similar functions. Hope this bot can help developers to filter useful issues.

### CODE REVIEW

File: _apps/close-issue.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+screenshots:
+- https://raw.githubusercontent.com/offu/close-issue-bot/master/screenshot.png
+authors: [ helloqiu, whtsky ]
+repository: offu/close-issue-bot
+host: https://damp-citadel-91689.herokuapp.com
+---
+[![Build Status](https://travis-ci.org/offu/close-issue-bot.svg?branch=master)](https://travis-ci.org/offu/close-issue-bot)
+[![codecov](https://codecov.io/gh/offu/close-issue-bot/branch/master/graph/badge.svg)](https://codecov.io/gh/offu/close-issue-bot)  
+## What It Does
+The bot will check new opened and reopened issues if they include some specific contents. Issues not passed will be commented then closed.
```
Comment (hiimbex) [inline _apps/close-issue.md]: We'd rather you not use 'bot' and instead use 'app' here.

File: _apps/close-issue.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+## Usage
+1. Create a ```/.github/issue-close-bot.yml``` file in your repo. Here's an example:
```
Comment (hiimbex) [inline _apps/close-issue.md]: ```/.github/issue-close-bot.yml``` -> `.github/issue-close-bot.yml` (also change from triple to single backticks please)

File: _apps/close-issue.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+``` yaml
+# Comment that will be sent if an issue is judged to be closed
+comment: test
```
Comment (hiimbex) [inline _apps/close-issue.md]: I would love if you could flesh out these examples more so it's easier for folks to understand

File: _apps/close-issue.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+host: https://damp-citadel-91689.herokuapp.com
+---
+[![Build Status](https://travis-ci.org/offu/close-issue-bot.svg?branch=master)](https://travis-ci.org/offu/close-issue-bot)
+[![codecov](https://codecov.io/gh/offu/close-issue-bot/branch/master/graph/badge.svg)](https://codecov.io/gh/offu/close-issue-bot)  
+## What It Does
+The bot will check new opened and reopened issues if they include some specific contents. Issues not passed will be commented then closed.
+## Usage
+1. Create a ```/.github/issue-close-bot.yml``` file in your repo. Here's an example:
+``` yaml
+# Comment that will be sent if an issue is judged to be closed
```
Comment (hiimbex) [inline _apps/close-issue.md]: Also can you specify if this file is required and if there are any default settings it requires to work?

**Comment by helloqiu** [comment]
@hiimbex Thx for your review. :)
I make some updates according to your suggestions. This app actually does not match the issue template directly. It closes issues according the **keywords** in the config file, so users need to setup what keywords they want.

**Comment by hiimbex** [comment]
Yea, that build fail was due to me breaking one of my apps. One other thing before I can merge is that we require all apps on the site to include a Code of Conduct. See https://github.com/probot/probot.github.io/blob/master/CONTRIBUTING.md#requirements-for-apps for details

**Comment by helloqiu** [comment]
@hiimbex Hi. I add a `Code of Conduct` in our repo. Check it here:
https://github.com/offu/close-issue-app/blob/master/CODE_OF_CONDUCT.md

**Comment by hiimbex** [comment]
Awesome! Thank you for adding that :)

**Pull request merged**